server/index.js

```diff
@@ -173,7 +177,14 @@ export default class Server {
 
       '/_next/:buildId/page/:path*': async (req, res, params) => {
         const paths = params.path || ['']
-        const page = `/${paths.join('/')}`
+        let page = `/${paths.join('/')}`
+
+        // TODO: This should reroute the path to a static page + query, but
+        // thus far, it won't load the new route or replace the component.
+        // The script does appear to load successfully though.... curious.
+        if (this.getStaticPage(page)) {
+          page = this.getStaticPage(page).page
+        }
```

Comment:
This appears to work by rerouting the path (if it matches a statically exported path) to its appropriate page template and query params. When in dev mode and requesting a page (after the initial page load) the script does appear to be sent from the server and run, however the new component does not render on the page, nor does the route update.